mailer.js

```diff
@@ -263,6 +255,52 @@ module.exports = function (log) {
     })
   }
 
+
+  Mailer.prototype.authorizeLoginEmail = function (message) {
+    log.trace({ op: 'mailer.authoirzeLoginEmail', email: message.email, uid: message.uid })
+
+    var templateName = 'authorizeLoginEmail'
+    var query = {
+      code: message.code,
```

Comment:
@seanmonstar - do these three parameters work for you? Would `code` be better as `unblockCode`?